tox.ini

```diff
@@ -1,6 +1,16 @@
 [tox]
-envlist = py27
+envlist = py27-django1{8,9}, flake8
+
+[flake8]
+max-line-length = 99
 
 [testenv]
-deps = Django==1.8
+deps =
+    django18: Django==1.8
+    django19: Django==1.9
```

Comment:
I think we actually want `=~` here. Django doesn't do a `.0` release, so `1.9` is equivalent to `1.9.0`